pandas/core/generic.py

```diff
@@ -72,6 +86,19 @@
 from pandas.io.formats.printing import pprint_thing
 from pandas.tseries.frequencies import to_offset
 
+if TYPE_CHECKING:
+    from pandas import Series, DataFrame
```

Comment:
I would put these in _typing

Reply:
the `errors` is local to this module since I think a global errors should also include `coerce`?

similarly a global `How` should probably be left, right, inner, outer?

I could rename them if you think these have additional utility, to say IgnoreRaise and FirstLast